lisp/init-utils.el

```diff
@@ -34,6 +34,50 @@
   (require 'init-const)
   (require 'init-custom))
 
+
+;; Handy file manipulation functions
```

Comment:
Can you please split this to another PR?
These can be put in `init-funcs.el`.

Reply:
Why don't you accept it then do the move? It might be faster. :)

Reply:
I don't think it's a good idea submit several features in one PR. It's hard to manage especially when it breaks the build.